include/boost/beast/http/impl/file_body_win32.hpp

```diff
@@ -156,6 +160,11 @@ struct basic_file_body<file_win32>
             auto const nread = body_.file_.read(buf_, n, ec);
             if(ec)
                 return boost::none;
+            if (nread == 0)
```

Comment:
what if `nread > 0` but `nread` is also less than what we expect in terms of reading everything in the file?

Reply:
Then we'll get the short read error in the next call to get.

This behavior is similar to POSIX read and fread.

Reply:
agree. no special handling required here.